app/Models/Mismatch.php

```diff
@@ -35,4 +35,16 @@ public function importMeta()
     {
         return $this->belongsTo(ImportMeta::class, 'import_id')->withDefault();
     }
+
+    /**
+     * Set the mismatch's item_id alongside the statement_guid
+     *
+     * @param  string  $value
+     * @return void
+     */
+    public function setStatementGuidAttribute($value)
```

Comment:
Actually, it took me some time to realise that setting the item_id cannnot be done in a `setItemIdAttribute()` mutator. That's because you would have to call that mutator explicitly by setting item_id to something, if even an empty value. That would be bad, so here you go: `setStatementGuidAttribute()` also sets the item_id.

Reply:
I mean to me that makes sense. I always read those setters in my mind with an `on` at the beginning of the method name. something like `onSetStatementGuid()`